nitro-protocol/contracts/libraries/signature-logic/ShortcuttingTurnTaking.sol

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+pragma experimental ABIEncoderV2;
+
+import {ExitFormat as Outcome} from '@statechannels/exit-format/contracts/ExitFormat.sol';
+import {NitroUtils} from '../NitroUtils.sol';
+import '../../interfaces/INitroTypes.sol';
+
+/**
+ * @dev Signatures in `sigs` part of `SignedVariablePart` must be in ascending order relative to participant index, which has created the signature.
+ */
+library ShortcuttingTurnTaking {
```

Comment:
I think it would be great to add a specific signature library (alongside `StrictTurnTaking` and `ShortcuttingTurnTaking`) for the Consensus app (that is "everyone signs one state" mode). This will resemble the fact that there are 3 different signature modes (the ones I just mentioned).

I don't believe that we should simplify the code in `ShortcuttingTurnTaking` if we introduce `Consensus`. On the contrary, it is great that we have some example code that can be used by developers. Maintenance of this code should not take much time as well as writing tests for it.

What do you think on this two points?

Reply:
I think you are right, we can add `Consensus` signature mode and keep the existing modes. Then over time it will be clear which ones end up being used.